scripts/computeCLBands.C

```diff
@@ -288,6 +331,74 @@ void computeCLBands(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int
     cout << sv2sigmaR[imass] << (imass<nmass-1? "," : "");
   cout << "};" << endl;
 
+  // compute the branon translation factor for each DM mass for branon
+  Double_t bt0sigma[nmass];
+  Double_t bt1sigmaL[nmass];
+  Double_t bt2sigmaL[nmass];
+  Double_t bt1sigmaR[nmass];
+  Double_t bt2sigmaR[nmass];
+  if(!channel.CompareTo("branon",TString::kIgnoreCase))
+    {
+      // initialize the array of translation factors for the brane tension limits
+      Double_t braneTensionVal[nmass];
+      for(Int_t imass=shift;imass<nmass;imass++)
+        {
+          Float_t mdm = massval[imass];
+          // release the memory of channelval and brval
+          delete [] channelval;
+          delete [] brval;
+          // number of considering channels
+          nChannels = 9;
+          channelval = new TString[nChannels];
+          brval = new Double_t[nChannels];
+          // call the function that computes the branching ratios and save them in channelval and brval
+          compute_branonBR(mdm,nChannels,channelval,brval,braneTensionVal[imass]);
+        }
+
+      // To translate the sigmav values from [cm^3 s^-1] to [GeV^-2], the sigmav limits have to be divided by
+      // h_bar^2 c^3 = (6.58*10^-25 [GeV s])^2 * (3.0*10^10 [cm s^-1])^2 = 1.17*10^-17 [GeV^2 cm^3 s^-1].
+      // The 0.001 factor is translating the brane tension limit from GeV to TeV.
```

Comment:
To take advantage of all the newly fixed constant :-) may I suggest something like:

// To translate sigmav values in [cm^3 s^-1] into brane tension values in [GeV^2], the following multiplication factor has to be applied
// h_bar^2 c^3 with dimension [GeV^2 s^2] * [cm^3 s^-3] = [GeV^2 cm^3 s^-1] with
// h = 6.626 07015×10^−34 J s ; e = 1.602176634×10^−19 C;  c = 299792458x10^2 cm s^−1
// following the values provided by the Particle Data Group in
// http://pdg.lbl.gov/2019/reviews/rpp2018-rev-phys-constants.pdf
// The final conversion formula also contains:
// - a factor (0.001) to translate the brane tension limit from GeV to TeV
// - a power 1/8 to relate sigma to f (see equation 7 of https://arxiv.org/abs/hep-ph/0302041)

Then you can compute once for all precisely this factor by doing:
factor_name = ((h/(2.*TMath:Pi())) / (e*10^9)) * (299792458x10^2)^3
and use it everywhere it's needed in the following lines

If you agree with this comment, it should be applied everywhere in the code where it is used.

Reply:
The sigmav values are in  [cm^3 s^-1] and with dividing h_bar^2 c^3 we are getting the **sigmav** values in [GeV^-2]. The brane tension has units of [GeV].

Having a variable for the defined/pre-computed "exact" constant is a good point. Since the Particle Data Group provided us with h_bar = 6.582119569x10^-22 [MeV s] = 6.58211956910x^-25 [GeV s] and c = 299792458.0 [m s^−1] = 299792458.0x10^2 [cm s^−1] we can just use these values and pre-compute h_bar^2 c^3 = 1.167329990x10^-17 [GeV^2 cm^3 s^-1]. I think there is no need to start the calculation from h in [J s]. Just to keep it as simple as possible.